jira.go

```diff
@@ -169,10 +169,11 @@ func openJiraTicket(flags flags, projectInfo jsn.Json, vulnForJira interface{},
 			Summary:     jiraTicket.Fields.Summary,
 			Description: jiraTicket.Fields.Description,
 		}
+		return nil, ticketFile, errors.New("*** WARN *** Skipping opening a ticket in --dryRun mode"), jiraApiUrl
 	}
 
 	// check that vulnId exist and dryRun is off
-	if len(vulnID) != 0 && !flags.optionalFlags.dryRun {
+	if len(vulnID) != 0 {
```

Comment:
why take the dryRun off? is it just somewhere else?

Reply:
Because we are now returning in the `dryRun` check above, please do confirm it does what I think :D

Reply:
ah yes sorry.